src/Grpc/HttpApi/test/testassets/IntegrationTestsWebsite/IntegrationTestsWebsite.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Aren't implicit `using`s on by default in C# projects everywhere in the the repo❔